src/mui-theme.HOC.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AppTheme from './theme';
+
+export default component => {
```

Comment:
I'm not too familiar with how this app has configured material ui. What's the difference between this and the MuiThemeProvider provided by material-ui/styles/MuiThemeProvider? It's almost the same as the original source: https://github.com/mui-org/material-ui/blob/v0.x/src/styles/MuiThemeProvider.js

I find it strange that multiple components need to add this to the context. Should be enough with a provider at the root-level of the app.

Reply:
That's a good questions.
I have to look into that and see if we can simply use the Provider on a higher leven and then get rid of the MuiTheme HOC completely, would be awesome